graphql/src/schema/meta.graphql

```diff
@@ -55,8 +55,8 @@ type _Block_ {
   hash: Bytes
   "The block number"
   number: Int!
-  "Timestamp of the block if available, format depends on the chain" 
-  timestamp: String
+  "Timestamp of the block if available" 
+  timestamp: Int
```

Comment:
The comment should also say something about how to interpret that number - maybe add `Integer representation of the timestamp stored in blocks for the chain` to tell people that we don't do any other processing